From 9360dab604647fad0492739eed47593768c84508 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 17 Feb 2021 14:58:07 +0800
Subject: [PATCH] Revert "irqchip/gic-v3-its: set flag
 "ITS_FLAGS_SAVE_SUSPEND_STATE" if rk3568 or rk3566"

This reverts commit 330a3597673cb9dc2c646d91908cd451717ff2b8.

Relpaced by commit 0c486401a3db ("irqchip/gic-v3-its: Unconditionally save/restore the ITS state on suspend")

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/irqchip/irq-gic-v3-its.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/irqchip/irq-gic-v3-its.c b/drivers/irqchip/irq-gic-v3-its.c
index b72070d0df20..a6b0fde29fb4 100644
--- a/drivers/irqchip/irq-gic-v3-its.c
+++ b/drivers/irqchip/irq-gic-v3-its.c
@@ -3598,9 +3598,6 @@ static int __init its_probe_one(struct resource *res,
 	if (GITS_TYPER_HCC(typer))
 		its->flags |= ITS_FLAGS_SAVE_SUSPEND_STATE;
 
-	if (of_machine_is_compatible("rockchip,rk3568") || of_machine_is_compatible("rockchip,rk3566"))
-		its->flags |= ITS_FLAGS_SAVE_SUSPEND_STATE;
-
 	err = its_init_domain(handle, its);
 	if (err)
 		goto out_free_tables;
-- 
2.35.3

